flux-cli/src/dist/conf/log4j2.properties

```diff
@@ -6,7 +6,7 @@ appender.console.name = STDOUT
 appender.console.type = Console
 appender.console.layout.type = PatternLayout
 #appender.console.layout.pattern = %d{yy/MM/dd HH:mm:ss} [%tn] %p %c: %m%n
-appender.console.layout.pattern = %d{yy/MM/dd HH:mm:ss} %c: %m%n
+appender.console.layout.pattern = %d{MM-dd-yyyy HH:mm:ss} %c: %m%n
```

Comment:
personally, I like yyyy-MM-dd

Reply:
Funny, that's actually what I meant to type out.